JustSaying/AwsTools/JustSayingConstants.cs

```diff
@@ -2,70 +2,70 @@ namespace JustSaying.AwsTools
 {
     public static class JustSayingConstants
     {
-        public const string ATTRIBUTE_REDRIVE_POLICY = "RedrivePolicy";
-        public const string ATTRIBUTE_ARN = "QueueArn";
-        public const string ATTRIBUTE_RETENTION_PERIOD = "MessageRetentionPeriod";
-        public const string ATTRIBUTE_VISIBILITY_TIMEOUT = "VisibilityTimeout";
-        public const string ATTRIBUTE_DELIVERY_DELAY = "DelaySeconds";
-        public const string ATTRIBUTE_POLICY = "Policy";
-        public const string ATTRIBUTE_ENCRYPTION_KEY_ID = "KmsMasterKeyId";
-        public const string ATTRIBUTE_ENCRYPTION_KEY_REUSE_PERIOD_SECOND_ID = "KmsDataKeyReusePeriodSeconds";
+        public const string AttributeRedrivePolicy = "RedrivePolicy";
```

Comment:
We can look at this separately, but some of these (where they're things we've defined rather than AWS) might be candidates for properties rather than constants, so that there's scope for drop-in upgrades rather than requiring recompilation, particularly where an application might have dependencies that have dependencies on JustSaying, meaning that an change in a newer version applied to a top-level app wouldn't take affect in the dependency without it also being recompiled.

Reply:
Changing to properties _should_ be transparent to the consuming files. They won't need altering.